modules/nw-udn-limitations.adoc

```diff
@@ -9,7 +9,10 @@
 While user-defined networks (UDN) offer highly customizable network configuration options, there are limitations that cluster administrators and developers should be aware of when implementing and managing these networks. Consider the following limitations before implementing a user-defined network.
 
 //Check on the removal of the DNS limitation for 4.18 or 4.17.z.
-* *DNS limitations*: DNS lookups for pods resolve to the pod's IP address on the cluster default network. Even if a pod is part of a user-defined network, DNS lookups will not resolve to the pod's IP address on that user-defined network.
+* *DNS limitations*:
+** DNS for UDN pods is unsupported. The Kubernetes API (KAPI) and external DNS are supported. 
+** DNS lookups for pods resolve to the pod's IP address on the cluster default network. Even if a pod is part of a user-defined network, DNS lookups will not resolve to the pod's IP address on that user-defined network.
```

Comment:
I think it is valuable to add this from the enhancement:

> DNS lookups for services and external entities will function correctly.

https://github.com/openshift/enhancements/blob/master/enhancements/network/user-defined-network-segmentation.md#dns